recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -56,46 +54,6 @@ def configure(self):
     def layout(self):
         basic_layout(self, src_folder="source")
 
-    @property
-    def _requires_compiler_rt(self):
-        return self.settings.compiler == "clang" and self.settings.build_type == "Debug"
-
-    def _test_for_compiler_rt(self):
```

Comment:
This horrible mess was added (by me) as part of https://github.com/conan-io/conan-center-index/pull/11436

I can't reproduce the issue with the conan docker images any more so I've removed it, mostly just to see what happens.

Reply:
d'oh.